docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-custom-pages-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='using-standalone-perspectives-standalone-custom-pages-proc']
+= Using standalone custom pages(dashboards)
+
+Apart from standalone perspectives, you can also embed custom pages (also known as dashboards) in your application. For accessing the custom pages from your application, provide the name of the custom page as the value of the `perspective` parameter. Note that the `perspective` parameter is case-sensitive.
```

Comment:
I just looked up the usage of parentheses in the IBM Style Guide. It says to use them for abbreviations, symbols, and measurements, but to avoid them in running text. 

So, I would reword as follows:
Apart from standalone perspectives, you can also embed custom pages, also known as dashboards, in your application.

I'm no editor, so we may want to ask Emily of someone if that it too many commas :)

Reply:
Changed as suggested.